### ISSUE TITLE
어떤 상황에서 어떤 Loss function을 주로 사용하는지

### ISSUE THREAD

**Issue by linkyouhj** [CUAI-CAU/PyTorch-Study-T2_2022#4]
<img width="835" alt="스크린샷 2022-03-24 오후 2 05 23" src="https://user-images.githubusercontent.com/55436953/159846033-34fb3e06-3aa4-4367-882f-028025fd003c.png">

링크에 더 잘 설명되어 있습니다!
[Loss&Optimizer](https://data-panic.tistory.com/20)

**Comment by kyoonkm** [comment]
헷갈렸는데 감사합니당 ~
